### ISSUE TITLE
Azure MYSQL Certificate file path is not found by Doctrine\DBAL\Driver\PDOConnection.php, even though it works on php artisan migrate

### ISSUE THREAD

**Issue by io-sar** [grokability/snipe-it#10530]
### Debug mode

- [X] I have enabled debug mode
- [X] I have read [checked the Common Issues page](https://snipe-it.readme.io/docs/common-issues)

### Describe the bug

Hi,

I am trying to set Snipe-it on docker container and connect it to an Azure MySql Flexible server.
I have followed the [instructions ](https://snipe-it.readme.io/docs/docker#using-a-standalone-database-not-a-container) and also some [old tickets](https://github.com/snipe/snipe-it/issues/7870) on the issue board.

In the end i did manage to have the application set up with the following .env settings

**# REQUIRED: DATABASE SETTINGS**

DB_CONNECTION=mysql
DB_HOST=<server>.mysql.database.azure.com
DB_DATABASE=snipeit
DB_USERNAME=<username>
DB_PASSWORD=<password>
DB_PREFIX=null
DB_DUMP_PATH='/usr/bin'
DB_CHARSET=utf8mb4
DB_COLLATION=utf8mb4_unicode_ci

**# OPTIONAL: SSL DATABASE SETTINGS**

DB_SSL=true
DB_SSL_IS_PAAS=true
DB_SSL_KEY_PATH=null
DB_SSL_CERT_PATH=null
DB_SSL_CA_PATH="./ssl/DigiCertGlobalRootCA.crt.pem"
DB_SSL_CIPHER=null

However even if `php artisan migrate` runs with no issue, when the app loads it states that:
"D'oh! Looks like we can't connect to your database. Please update your database settings in your .env file. Your database says: SQLSTATE[HY000] [2002] (SQL: select 2 + 2)"

With the following [Error from debugger.txt](https://github.com/snipe/snipe-it/files/7892141/Error.from.debugger.txt)

which points to me that the filepath of the certificate is not set correctly somewhere. (eg Doctrine\DBAL\Driver\PDOConnection.php)

I have the same issue from both my MacBook, Desktop and Docker container

### Reproduction steps

Prerequisite:
Azure MySQL Flexible server, the configuration, install dependencies, generate key are followed as the [installation steps](https://snipe-it.readme.io/docs/installation)
 
1. Start the server/ docker run / `php artisan serv`
2. One can see that the` php artisan migrate` will run successfully and the DB is populated
3.Once the migration is done, I will still have missconfigured db message, and the errors
PDOException: failed loading cafile stream: `./ssl/DigiCertGlobalRootCA.crt.pem' in C:\wamp64\www\snipe-it\vendor\doctrine\dbal\lib\Doctrine\DBAL\Driver\PDOConnection.php:39
as depicted on the screenshot.

### Expected behavior

The file path of the certificate will be configured correctly on the Doctrine\DBAL\Driver\PDOConnection.php, and the app will run without issues.

### Screenshots

![Example Capture](https://user-images.githubusercontent.com/22503601/150015062-212c353c-c1b9-4e9e-a248-93c8e932334d.PNG)

### Snipe-IT Version

v5.3.6

### Operating System

Windows, Mac, Docker

### Web Server

Apache

### PHP Version

7.4

**Comment by snipe** [comment]
Hi there - sorry for the freakishly long time in responding here. This one fell off our radar :( 

Since the time this issue was opened, a ton of changes have been made in Snipe-IT. If you're still seeing this issue in the current version, please open a new issue that includes:

- Version of Snipe-IT you're running
- Specific errors you're seeing in the UI and/or in the browser console and error logs
- Steps to reproduce 
- Whether or not you can reproduce this on the demo (https://demo.snipeitapp.com)

Thanks - and again, sorry for the very late reply. We're doing some cleanup of old issues, so we're rediscovering some older things that didn't get addressed the first time around. 

![BeakerEmbarrased](https://github.com/user-attachments/assets/68ca3f30-2f2c-4428-b905-3086d92b88b2)

<blockquote><img src="https://demo.snipeitapp.com/favicon.ico" width="48" align="right"><div><strong><a href="https://demo.snipeitapp.com/">Snipe-IT Asset Management Demo</a></strong></div></blockquote>
